ostree-ext/src/container/deploy.rs

```diff
@@ -122,6 +136,24 @@ pub async fn deploy(
             flags,
             cancellable,
         )?;
+
+        // We end up re-executing ourselves as a subprocess because
+        // otherwise right now we end up with a circular dependency between
+        // crates. We need an option to skip though so when the *main*
+        // bootc install code calls this API, we don't do this as it
+        // will have already been handled.
+        if !options.skip_completion {
+            // Note that the sysroot is provided as `.`  but we use cwd_dir to
+            // make the process current working directory the sysroot.
+            let st = Command::new("bootc")
```

Comment:
Maybe instead

```suggestion
            let st = Command::new(<argv0>)
```

So that people running a custom non-PATH binary of bootc (during dev / debug) don't get surprised when the re exec is a different binary altogether

Reply:
Good catch! Changed to use `/proc/self/exe`